lib/model/folder.go

```diff
@@ -1246,7 +1246,6 @@ func (f *folder) updateLocals(fs []protocol.FileInfo) {
 		"items":     len(fs),
 		"filenames": filenames,
 		"sequence":  seq,
-		"version":   seq, // legacy for sequence
```

Comment:
Please forgive me if I'm looking at the wrong thing, but it seems that SyncTrayzor relies on this.

https://github.com/canton7/SyncTrayzor/blob/1ec7e4fadd5e7e64fb56348de630b85ccab35f07/src/SyncTrayzor/Syncthing/ApiClient/FolderStatus.cs#L47-L48